code/modules/awaymissions/mission_code/ghost_role_spawners/golems.dm

```diff
@@ -134,6 +134,24 @@
 		user.death()
 		return
 
+/obj/effect/mob_spawn/human/golem/attackby(obj/item/slimepotion/transference/I, mob/user, params)
+	. = ..()
+	if(istype(src, /obj/effect/mob_spawn/human/golem/servant))
+		src.has_owner = FALSE
+	src.flavour_text = null
+	if(iscarbon(user) && can_transfer)
+		var/human_transfer_choice = alert("Transfer your soul to [src]? (Warning, your old body will die!)",,"Yes","No")
+		if(human_transfer_choice != "Yes")
+			return
+		if(QDELETED(src) || uses <= 0)
+			return
+		log_game("[key_name(user)] used [I] to transfer their mind into [src]")
+		user.visible_message("<span class='notice'>As [user] applies the potion on the golem shell, a faint light leaves them, moving to [src] and animating it!</span>","<span class='notice'>You apply the potion to [src], feeling your mind leave your body!</span>")
```

Comment:
Another styling thing but this might be neater if it was moved to two lines rather than one.
```suggestion
		user.visible_message("<span class='notice'>As [user] applies the potion on the golem shell, a faint light leaves them, moving to [src] and animating it!</span>",
			"<span class='notice'>You apply the potion to [src], feeling your mind leave your body!</span>")
```